release_constants.json

```diff
@@ -0,0 +1,43 @@
+{
+    "RELEASE_SUMMARY_FILEPATH": "../release_summary.md",
+    "CHANGELOG_HEADER": "### Changelog:\n",
+    "COMMIT_HISTORY_HEADER": "### Commit History:\n",
+    "EMAIL_HEADER": "### Email C&P Blurbs about authors:\n",
+    "EXISTING_AUTHORS_HEADER": "### Existing Authors:\n",
+    "ISSUES_HEADER": "### Issues mentioned in commits:\n",
+    "NEW_AUTHORS_HEADER": "### New Authors:\n",
+    "NEW_CONTRIBUTORS_HEADER": "### New Contributors:\n",
+    "CONSTANTS_PATH": "assets/constants.ts",
+    "SCHEMA_CONFIG_PATH": "feconf.py",
+    "INDEX_YAML_PATH": "index.yaml",
+    "GAE_DIR": "../oppia_tools/google-cloud-sdk-304.0.0/google-cloud-sdk/platform/google_appengine",
+    "GCLOUD_PATH": "../oppia_tools/google-cloud-sdk-304.0.0/google-cloud-sdk/bin/gcloud",
+    "PY_GITHUB_PATH": "../oppia_tools/PyGithub-1.45",
+    "CUT_BRANCH_MODULE_PATH": "scripts.release_scripts.cut_release_or_hotfix_branch",
+    "REPO_SPECIFIC_CHANGE_MODULE_PATH": "scripts.release_scripts.get_repo_specific_changes",
+    "ISSUE_URL_FORMAT_STRING": "https://github.com/oppia/oppia/issues/%s",
+    "SCHEMA_VERSIONS_TO_JOBS_MAPPING": {
+        "CURRENT_COLLECTION_SCHEMA_VERSION": "CollectionMigrationOneOffJob",
+        "CURRENT_STATE_SCHEMA_VERSION": "ExplorationMigrationJobManager",
+        "CURRENT_SKILL_CONTENTS_SCHEMA_VERSION": "SkillMigrationOneOffJob",
+        "CURRENT_MISCONCEPTIONS_SCHEMA_VERSION": "SkillMigrationOneOffJob",
+        "CURRENT_RUBRIC_SCHEMA_VERSION": "SkillMigrationOneOffJob",
+        "CURRENT_STORY_CONTENTS_SCHEMA_VERSION": "StoryMigrationOneOffJob",
+        "CURRENT_SUBTOPIC_SCHEMA_VERSION": "TopicMigrationOneOffJob",
+        "CURRENT_STORY_REFERENCE_SCHEMA_VERSION": "TopicMigrationOneOffJob"},
```

Comment:
For cleanliness, move the final `},` one line below and deindent it by 4.

Reply:
Done.